catroid/src/androidTest/java/org/catrobat/catroid/test/content/project/ProjectTest.java

```diff
@@ -109,11 +111,13 @@ public void testSetDeviceData() {
 		header.setCatrobatLanguageVersion(OLD_LANGUAGE_VERSION);
 		header.setApplicationName(OLD_APPLICATION_NAME);
 		header.setPlatform(OLD_PLATFORM);
+		header.setPlatformVersion(OLD_PLATFORM_VERSION);
 
 		project.setDeviceData(InstrumentationRegistry.getTargetContext());
 
 		assertEquals(Constants.CURRENT_CATROBAT_LANGUAGE_VERSION, header.getCatrobatLanguageVersion());
 		assertEquals(InstrumentationRegistry.getTargetContext().getString(R.string.app_name), header.getApplicationName());
 		assertEquals(Constants.PLATFORM_NAME, header.getPlatform());
+		assertEquals(String.valueOf(Build.VERSION.SDK_INT), header.getPlatformVersion());
```

Comment:
This test doesn't really test the change, but it was still missing imo. So i sneaked it in here :scream: